libs/libcommon/cmake/find_jemalloc.cmake

```diff
@@ -14,7 +14,7 @@
 
 option (ENABLE_JEMALLOC "Set to TRUE to use jemalloc" ON)
 # TODO: Make ENABLE_JEMALLOC_PROF default value to ON after https://github.com/pingcap/tics/issues/3236 get fixed.
-option (ENABLE_JEMALLOC_PROF "Set to ON to enable jemalloc profiling" OFF)
+option (ENABLE_JEMALLOC_PROF "Set to ON to enable jemalloc profiling" ON)
```

Comment:
Have we fixed this?

Reply:
It does not related to the former deadlock.

BTW, the former deadlock has not appeared again so far.

Reply:
Could we remove the comment?

Reply:
fixed